tests/integration/tables/keychain_items.cpp

```diff
@@ -14,34 +14,31 @@
 namespace osquery {
 namespace table_tests {
 
-class keychainItems : public testing::Test {
+class KeychainItemsTest : public testing::Test {
  protected:
   void SetUp() override {
     setUpEnvironment();
   }
 };
 
-TEST_F(keychainItems, test_sanity) {
-  // 1. Query data
+TEST_F(KeychainItemsTest, test_sanity) {
   auto const data = execute_query("select * from keychain_items");
-  // 2. Check size before validation
-  // ASSERT_GE(data.size(), 0ul);
-  // ASSERT_EQ(data.size(), 1ul);
-  // ASSERT_EQ(data.size(), 0ul);
-  // 3. Build validation map
-  // See helper.h for avaialbe flags
-  // Or use custom DataCheck object
-  // ValidationMap row_map = {
-  //      {"label", NormalType}
-  //      {"description", NormalType}
-  //      {"comment", NormalType}
-  //      {"created", NormalType}
-  //      {"modified", NormalType}
-  //      {"type", NormalType}
-  //      {"path", NormalType}
-  //}
-  // 4. Perform validation
-  // validate_rows(data, row_map);
+
+  ValidationMap row_map = {
+      {"label", NormalType},
+      {"description", NormalType},
+      {"comment", NormalType},
+      {"created", NormalType},
+      {"modified", NormalType},
+      {"type",
+       SpecificValuesCheck{"password",
+                           "certificate",
+                           "symmetric key",
+                           "public key",
+                           "private key"}},
+      {"path", NonEmptyString},
+  };
+  validate_rows(data, row_map);
```

Comment:
This should have an assert that it has rows. May or may not need to include test data

Reply:
I've added an assert which hopefully does this, thanks.